spec/controllers/chargeback_controller_spec.rb

```diff
@@ -26,7 +26,8 @@
       before do
         cbr = FactoryGirl.create(:chargeback_rate, :rate_type => "Storage")
         ChargebackRate.set_assignments(:Storage, [{:cb_rate => cbr, :tag => [tag, "vm"]}])
-        sandbox = {:active_tree => :cb_assignments_tree, :trees => {:cb_assignments_tree => {:active_node => 'xx-Storage'}}}
+        sandbox = {:active_tree => :cb_assignments_tree, :trees => \
+          {:cb_assignments_tree => {:active_node => 'xx-Storage'}}}
```

Comment:
@aljesusg it's still showing up as it was... Maybe you didn't push your changes?